kimsoohyun/app.js

```diff
@@ -41,12 +109,77 @@ app.post('/users', async (req, res) => {
                            password)
          VALUES (?, ?, ?, ?, ?);
         `, [id, name, email, profileImage, password]);
-    res.status(201).json({message: "userCreateed", users}
+    res.status(201).json({message: "user created"}
+    );
+})
+app.post('/posts', async (req, res) => {
+    const {id, imageUrl, title, content, userId} = req.body;
+    const posts = await appDataSource.query(
+        `INSERT INTO posts(id,
+                           image_url,
+                           title,
+                           content,
+                           user_id)
+         VALUES (?, ?, ?, ?, ?);`, [id, imageUrl, title, content, userId]
+    );
+    return res.status(200).json({"message": "post created"})
```

Comment:
```suggestion
    return res.status(201).json({"message": "post created"})
```
post method와 응답으로 돌아오는 message와 적절한 status code로 바꿔주세요~